--- RutilTv0.15/configure.sh.orig	2007-06-30 19:28:16.000000000 +0200
+++ RutilTv0.15/configure.sh	2007-11-25 09:59:29.000000000 +0100
@@ -18,7 +18,7 @@
 #
 #
 # Variables :
-prefix=/usr/local
+prefix=/usr
 launcher=
 helper_mode=4755
 makefile_target='$(RUTILT_NAME) $(HELPER_NAME)'
@@ -33,7 +33,7 @@
 ip_script_name=set_ip.sh
 ip_script_prefix=share/rutilt/
 helper_launcher_name=helper_launcher.sh
-helper_launcher_prefix=share/rutilt/
+helper_launcher_prefix=lib/rutilt/
 user_config_name=/rutilt/
 user_config_prefix=/.config/
 icon_prefix=share/rutilt/
